HIRING.md

```diff
@@ -47,27 +47,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
 - Have you faced any challenges when working with designers and/or backend developers? Could you tell us a bit more about your experience on collaborating with both?
 
 ## Assignment
-The candidate will be asked to make an assigment with a 24 hour deadline. He'll get to pick the best day to receive the details of the assignment then the clock will start ticking. The assignment will consist in making a simple app to display a list of items that comes from an API. The list should be able to be accessed offline.
+The candidate will be asked to make an assigment with a 24 hour deadline. They will be able to pick the best day to receive the details of the assignment before the clock starts ticking. The assignment will consist of making a simple app to display a list of items that come from an API.
 
 # Job description for an iOS position
 
 ### A bit about you
-- 01 Experience with building top level applications for Apple iOS/OS X, and started peeping into Watch OS and tvOS.
-- 02 Extensive experience with team and client work.
-- 03 A broad understanding of Swift, Core Data and Web-services.
-- 04 Self-sustaining and forever curious about how to improve your workflow.
-- 05 Experience with unit testing.
-- 06 Further user experience knowledge is a bonus.
-- 07 or collaborating on open source projects is a big plus.
+- 01 Familiarity with building applications for iOS and/or Mac OS. Maybe even some Watch and tvOS.
```

Comment:
Or maybe:
"You enjoy building applications for iOS"
Or something in that tone? The repetition of the word "Familiarity" (or Experience for that matter) annoys me a bit 😅